manifests/script-lab-local.outlook.xml

```diff
@@ -309,69 +309,69 @@
         <bt:String id="PG.TabLabel" DefaultValue="[DEV] Script Lab" />
         <bt:String id="PG.GroupLabel" DefaultValue="Script" />
         <bt:String id="PG.AboutGroupLabel" DefaultValue="About Script Lab">
-          <Override Locale="de" Value="Über Script Lab"/>
+          <bt:Override Locale="de" Value="Über Script Lab"/>
```

Comment:
Actually, for the local, we just decided to keep it always English (that's what we recently did in the main WXP one), so that folks don't have to localize twice.